form.spec

```diff
@@ -8,12 +8,12 @@ Requires: zlib
 %endif
 
 %prep
-%setup -q -n form-4.1
+%setup -q -n form-4.2.1
```

Comment:
@iarspider , please change this to `form-%{realversion}` so that next time we can just update the version in once place